javascript/javascript1/week3/homework/series-duration-ofmylife.js

```diff
@@ -0,0 +1,43 @@
+const seriesDurations = [
+    {
+        title: "Game of thrones",
+        days: 3,
+        hours: 1,
+        minutes: 0,
+    },
+    {
+        title: "Friends",
+        days: 4,
+        hours: 2,
+        minutes: 20,
+    },
+    {
+        title: "Breaking Bad",
+        days: 6,
+        hours: 0,
+        minutes: 50,
+    }
+];
+  
+const lifeSpanInMinutes = 80 * 525600;
+
+function logOutSeriesText() {
+    let totalSeriesDuration = 0;
+
+    for(let i = 0; i < seriesDurations.length; i++){
+        if (seriesDurations[i] == null) {
+            console.log("Error!");
```

Comment:
"Error!" does not tell me anything. Make it a more descriptive error message to help the person using your code.